lib/angular/sync-ng.js

```diff
@@ -22,7 +22,16 @@ angular.module('wfm.sync.service', [])
   }
 
   syncService.init = function($fh, syncOptions) {
-    sync.init($fh, syncOptions);
+    $fh = $fh || {};
+    syncOptions = syncOptions || {};
+
+    //If the feedhenry API is passed, use this.
+    //Otherwise default to the global window.$fh
+    if ($fh.sync) {
+      sync.init($fh, syncOptions, mediator);
```

Comment:
Probably easier just to call this using `sync.init($fh || widow.$fh, syncOptions, mediator);` A little less repeating.